cmd/asset-syncer/postgresql_utils.go

```diff
@@ -187,7 +244,7 @@ func (m *postgresAssetManager) filesExist(chartFilesID, digest string) bool {
 func (m *postgresAssetManager) insertFiles(chartFilesID string, files models.ChartFiles) error {
 	query := fmt.Sprintf(`INSERT INTO %s (chart_files_ID, info)
```

Comment:
so yes, we need to either add the namespace to `chart_files_ID` or add the namespace as a different column.

Reply:
Right, I went with the latter (namespaces as a different column), as I think we use the `chart_files_ID` in the UI which would look a bit odd as `kubeapps/bitnami/dex-1.8.0`.